packages/components/Shimmer/ios/ShimmerContainerView.swift

```diff
@@ -0,0 +1,144 @@
+import Foundation
+import FluentUI
+
+
+@objc(MSFShimmerContainerView)
+class ShimmerContainerView: UIView {
```

Comment:
I'm not sure I quite understand the reason we need this file to have all this logic. I was under the impression that all this logic should live in FluentUI-Apple and this class would simply extend that control and just pass the values through to that.

Reply:
The FUA Control only wants to Shimmer the top level subviews. I could never actually get it to shimmer with any test code I wrote. This code is lifted from fabric-react-native and converted to swift, comments and everything. 
It's inverting the logic to shimmer the leaf node / children subviews, which I guess works better with React Native's View heirarchy of nested children views.

Reply:
then extend the FU-A to make it easier for everyone?

Reply:
I'm not sure it would provide value to FUA / it goes against how it was orginally implemented?? I'm still a little unclear on usage of Shimmer in production is, besides the demoes in our test app

Reply:
I guess the thing here is that now the FUA control differs from the FURN control and I don't think we want that?